app/views/handlers/question.py

```diff
@@ -12,11 +13,54 @@ class Question(BlockHandler):
     def rendered_block(self):
         return self._render_block(self.block['id'])
 
-    def get_context(self, form):
-        return build_question_context(self.rendered_block, form)
+    def get_next_location_url(self):
+        answer_action = self._get_answer_action()
+        if self._has_redirect_to_list_add_action(answer_action):
+            location_url = self._get_list_add_question_url(answer_action['params'])
 
-    def handle_post(self, form):
-        self.questionnaire_store_updater.update_answers(form)
+            if location_url:
+                return location_url
+
+        return self.router.get_next_location_url(
+            self._current_location, self._routing_path
+        )
+
+    @staticmethod
```

Comment:
Do we have a style guide for static methods? Usually I prefer them in one block at the top of the class's method definitions

Reply:
I don't think we have a style guide. But have moved to the top of the class.